Author: Bernhard M. Wiedemann <bwiedemann suse de>
Date: 2018-08-18
https://bugzilla.opensuse.org/show_bug.cgi?id=1100677

Disable CPU detection to always use SSE3

It should have used SSE2 for wider hardware support, but
https://github.com/jgaeddert/liquid-dsp/issues/137 prevented it

Index: liquid-dsp-1.3.1.git1508199169.4c9a7514/scripts/ax_ext.m4
===================================================================
--- liquid-dsp-1.3.1.git1508199169.4c9a7514.orig/scripts/ax_ext.m4
+++ liquid-dsp-1.3.1.git1508199169.4c9a7514/scripts/ax_ext.m4
@@ -62,7 +62,7 @@ AC_DEFUN([AX_EXT],
 
  AC_CACHE_CHECK([whether mmx is supported], [ax_cv_have_mmx_ext],
   [
-    ax_cv_have_mmx_ext=no
+    ax_cv_have_mmx_ext=yes
     if test "$((0x$edx>>23&0x01))" = 1; then
       ax_cv_have_mmx_ext=yes
     fi
@@ -70,7 +70,7 @@ AC_DEFUN([AX_EXT],
 
  AC_CACHE_CHECK([whether sse is supported], [ax_cv_have_sse_ext],
   [
-    ax_cv_have_sse_ext=no
+    ax_cv_have_sse_ext=yes
     if test "$((0x$edx>>25&0x01))" = 1; then
       ax_cv_have_sse_ext=yes
     fi
@@ -78,7 +78,7 @@ AC_DEFUN([AX_EXT],
 
  AC_CACHE_CHECK([whether sse2 is supported], [ax_cv_have_sse2_ext],
   [
-    ax_cv_have_sse2_ext=no
+    ax_cv_have_sse2_ext=yes
     if test "$((0x$edx>>26&0x01))" = 1; then
       ax_cv_have_sse2_ext=yes
     fi
@@ -86,7 +86,7 @@ AC_DEFUN([AX_EXT],
 
  AC_CACHE_CHECK([whether sse3 is supported], [ax_cv_have_sse3_ext],
   [
-    ax_cv_have_sse3_ext=no
+    ax_cv_have_sse3_ext=yes
     if test "$((0x$ecx&0x01))" = 1; then
       ax_cv_have_sse3_ext=yes
     fi
@@ -96,7 +96,7 @@ AC_DEFUN([AX_EXT],
   [
     ax_cv_have_ssse3_ext=no
     if test "$((0x$ecx>>9&0x01))" = 1; then
-      ax_cv_have_ssse3_ext=yes
+      ax_cv_have_ssse3_ext=no
     fi
   ])
 
@@ -104,7 +104,7 @@ AC_DEFUN([AX_EXT],
   [
     ax_cv_have_sse41_ext=no
     if test "$((0x$ecx>>19&0x01))" = 1; then
-      ax_cv_have_sse41_ext=yes
+      ax_cv_have_sse41_ext=no
     fi
   ])
 
@@ -112,7 +112,7 @@ AC_DEFUN([AX_EXT],
   [
     ax_cv_have_sse42_ext=no
     if test "$((0x$ecx>>20&0x01))" = 1; then
-      ax_cv_have_sse42_ext=yes
+      ax_cv_have_sse42_ext=no
     fi
   ])
 
@@ -120,7 +120,7 @@ AC_DEFUN([AX_EXT],
   [
     ax_cv_have_avx_ext=no
     if test "$((0x$ecx>>28&0x01))" = 1; then
-      ax_cv_have_avx_ext=yes
+      ax_cv_have_avx_ext=no
     fi
   ])
 
